modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -25,7 +25,10 @@ test.describe('Manage object fields through Model Builder', () => {
 			await apiHelpers.listTypeAdmin.postRandomListTypeDefinition();
 
 		const objectDefinition =
-			await apiHelpers.objectAdmin.postRandomObjectDefinition({objectFolderExternalReferenceCode: 'default'});
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				objectFolderExternalReferenceCode: 'default',
+				status: {code: 0},
+			});
```

Comment:
Sorry if it was confusing, I've always sent changes like this one(change function and then updated it's usages) in different commits cause I thought it was easier to understand. But I will take that into consideration next time, thanks!! 💟

Reply:
as i explained to you in person, there is no problem in changing the props in one commit and making the function changes in another commit! 

i was confused because you changed the props and also changed the function on the same commit, but without adding the new prop you've created. so no worries, you are not doing wrong, is just a matter to organize certain changes better! :sunflower: